### DISCUSSION TITLE
Match the current with the deprecated command line flags

### DISCUSSION THREAD

**Discussion by mynanshan** [ANTsX/ANTs#1789] [Q&A] [answered]
I am looking into a years-old program, where the flags for antsRegistration are `-m`, `-t`, `-s`, etc. I tried to figure out what the arguments are, but only found on  [the current wiki page](https://github.com/ANTsX/ANTs/wiki/Anatomy-of-an-antsRegistration-call) a list of arguments such as `--metric`, `--transform` and `--shrink-factors`. Existing pieces of information on the former arguments are incomplete, and links to older documentation by stnava are redirected to the current repo. In general, It seems I still have to match the flags by guessing. 

Does anyone know where to find full documentation on the abbreviated flags?

**Comment by cookpa** [comment, marked as answer]
`antsRegistration --help` should list both

**Reply by mynanshan** [reply]
Thanks. I noticed a `-l` flag is used in that program but has not been included in the help page. Is it a deprecated flag?

**Reply by cookpa** [reply]
Yes, that flag was left over from an older set of optimizers, replaced circa 2013. It remained in the code until 2022 but it didn't affect the results.
